ipaserver/install/installutils.py

```diff
@@ -471,16 +536,21 @@ def get_host_name(no_host_dns):
     verify_fqdn(hostname, no_host_dns)
     return hostname
 
+
 def get_server_ip_address(host_name, unattended, setup_dns, ip_addresses):
     hostaddr = resolve_ip_addresses_nss(host_name)
     if hostaddr.intersection(
             {ipautil.UnsafeIPAddress(ip) for ip in ['127.0.0.1', '::1']}):
-        print("The hostname resolves to the localhost address (127.0.0.1/::1)", file=sys.stderr)
-        print("Please change your /etc/hosts file so that the hostname", file=sys.stderr)
-        print("resolves to the ip address of your network interface.", file=sys.stderr)
-        print("The KDC service does not listen on localhost", file=sys.stderr)
-        print("", file=sys.stderr)
-        print("Please fix your /etc/hosts file and restart the setup program", file=sys.stderr)
+        err_msg = (
```

Comment:
Unrelated change, please revert. This will create conflicts with another pending patch.